vagrant-kubernetes-istio/setup_kubectl_config_host.sh

```diff
@@ -1,10 +1,19 @@
 #!/bin/bash
 
 # Setting up kubectl on host to talk to kubernetest cluster on Vagrant VM.
-echo "your old ~/.kube/config file can be found at ~/.kube/config_old"
-ls ~/.kube/config_old
+
+# Save and unset KUBECONFIG in case users set it pointing to a k8s cluster
+export KUBECONFIG_SAVED=$KUBECONFIG
+unset KUBECONFIG
+
+# Set kube config file on host
+ls ~/.kube/config_old > /dev/null
 if [ $? -ne 0 ]; then
-    cp ~/.kube/config ~/.kube/config_old
+	ls ~/.kube/config > /dev/null
```

Comment:
nit: indentation if off here..

Reply:
I always have indentation problem after switching to eclipse IDE... :(